api/objects.proto

```diff
@@ -249,6 +249,14 @@ message Cluster {
 	// If the key is empty, the node will be unlocked (will not require a key
 	// to start up from a shut down state).
 	repeated EncryptionKey unlock_keys = 9;
+	
+	// RoutingMeshOff denotes whether the routing mesh is currently disabled.
+	// By default the routing mesh is enabled. As result of user removing the
+	// ingress network, this flag will be set to true. Cluster needs to carry
+	// this information, so that on reboot or leader re-election, new leader
+	// will not automatically create the routing mesh if it cannot find one
+	// in store. 
+	bool routingMeshOff = 10;
```

Comment:
I am happy to avoid the extra state, but the other options I had before were also sort of hacky, like to add a mock ingress network object to store, to denote the ingress nw need not to be created. 

IIUC, the allocator would attempt to create the default cluster object, the failure will indicate this is a leader reelection, or a cluster start after a shutdown. In that case I can 
```
if ErrNoIngress && cannot create cluster object {
  routingMeshDisabled == true
}
```
But what does it mean that allocator succeeds in creating the default cluster object ?
Once allocator gets a chance to run, isn't the default cluster obj already present in store, as created by the manager ? Or is the allcoator cluster object a mock one not seen by anybody else ?